packaging/opensuse-42.2/snapd.spec

```diff
@@ -32,8 +32,8 @@
 
 %define systemd_services_list snapd.refresh.timer snapd.refresh.service snapd.socket snapd.service snapd.autoimport.service snapd.system-shutdown.service
 Name:           snapd
-Version:        2.27.5
-Release:        1
+Version:        2.27.6
+Release:        0
```

Comment:
I think this says at 1

Reply:
Oh? It is "0" for fedora? Happy to change it back though.

Reply:
It should always be zero.